### ISSUE TITLE
Impossible to set PhoneNumberField to None/null

### ISSUE THREAD

**Issue by jambonrose** [stefanfoulis/django-phonenumber-field#102]
tl;dr: Given the current implementation of the field, it is impossible to set the field to the database `null` value.

In the current implementation, an empty `PhoneNumberField` field will *always* be set to the empty string. While Django recommends against using subclasses of `CharField` with the option `null=True`, this sometimes becomes necessary when the field is both optional and unique.

    phone = PhoneNumberField(blank=True, default=None, null=True, unique=True)

In the case of an optional unique field, it becomes important to avoid the empty string, as this causes uniqueness `IntegrityError` to be raised. Instead, the use of `null` (or `None` in Python) become necessary, as `null` values do not cause uniqueness conflicts.

It is therefore important for the field to be able to support the ability to be set to `None` or `null`.

**Comment by Timmerop** [comment]
I'm having this same issue. Looks like the original fix has been overwritten here: https://github.com/stefanfoulis/django-phonenumber-field/commit/036ebca675f554d1a40be060894e21fdda7046da#diff-497bc1cf2ee9e2f3444d450019a283c4

**Comment by mcniac** [comment]
any updates on this? why was this change removed?
